javacutil/src/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -347,6 +347,13 @@ public static Type wildLowerBound(ProcessingEnvironment env, TypeMirror tm) {
             return TypeAnnotationUtils.unannotatedType(t);
         }
     }
+
+    /** Returns the {@link TypeMirror} for Object. */
```

Comment:
Let's expand the `Object` to `java.lang.Object` to prevent confusion.

Reply:
There is an `isObject` method already. Should this method be just `getObject`? Would it help to keep these two methods next to each other?